src/io_hid_keys.rs

```diff
@@ -0,0 +1,15 @@
+// exports from <IOKit/hid/IOHIDKeys.h>
+
+use types::IOOptionBits;
+
+pub type IOHIDReportType = IOOptionBits;
```

Comment:
I think this one needs to be `c_int`. `IOHIDReportType` is used in the function signature for `IOHIDReportCallback`, so changing this type could affect the ABI, which is dangerous.